### DISCUSSION TITLE
Moving here was a bad idea for the broader community

### DISCUSSION THREAD

**Discussion by JulianOliver** [meshtastic/Discussions#93] [General]
Realise it's a bit late on this but I fear moving discussions here was not a good idea as regards the broader project of Meshtastic uptake by the general public. In my region Meshtastic is seeing growth among non-technical sorts and GitHub just isn't going to work for them I think. We developers can sometimes forget how intimidating the likes of GitHub can be, whereas Discourse is already quite well known. Some non-technies are enthusiastic about MT as a novelty, a new way of staying in touch, while others are getting involved as a practical resilience fallback in the event of a very real event, like a natural disaster. 

IMO we ought to reboot the Discourse forum, which is far more welcoming and intuitive, such that people with little technical interest or background can be made welcome and get the support they need. 

GitHub is also about as centralised as it gets, which is somewhat contrary to the spirit of decentralisation, but that's another thing!

Could we not combine these needs worlds with an integration of GitHub in Discourse?

https://meta.discourse.org/t/discourse-code-review/103142

**Comment by thebentern** [comment]
For non-technical users the Discord server and Subreddit cover these same needs (and then some). Incidentally, they are also the two biggest communities we have.

**Reply by JulianOliver** [reply]
True. Both are pretty poor for structured conversation though. Discord is also for symmetric communication, which is not so easy across timezones, people that do not type quickly or have English as their 2nd language. So easy to get lost in the river of chat. Discourse was working so well, slow paced, self-archiving and threaded by topic.

**Reply by thebentern** [reply]
Discord actually has a new-ish feature Threads where you can post message board style as well. We use it extensively under some of the different parent topics.
<img width="1145" alt="image" src="https://github.com/user-attachments/assets/4bcf080f-dc90-4ae9-8472-fddb5e447166">

**Reply by JulianOliver** [reply]
Put another way, the format of the forum is the biggest advantage of Discourse. Disccord (which is group chat) or a 'subreddit' is a hard sell to a 70yr old sheep farmer, or a technology averse older geography teacher - to think of a couple of examples. Forums not so, as they were the goto for decades, have a history of use across those generations.

Safer for youth too - Discord is a minefield for child safety.

And while indeed we have before us a discussion forum right here, asymmetric, structured, the landing page for GitHub announces itself as:

> Build and ship software on a single, collaborative platform. Join the world’s most widely adopted AI-powered developer platform.

Most looking for a forum to ask totally normal questions about, say, power, range, weather proofing, the app, will get there and go "nope, not for me".

**Reply by thebentern** [reply]
One of the main reasons why we closed Discourse was the complete lack of engagement there after everyone moved to the other communities. It became largely unmoderated by our community leaders because of this and slowly turned into a haven for bad advice for users. It had a good run for a while but became a pretty depressing shell of its former glory. As harsh as it sounds, we simply can't cater to every audience, and I would rather narrow the focus of our community leaders to supporting the platforms that folks actually use.
